### DISCUSSION TITLE
Possible cooperation on alternative opensource BPMS

### DISCUSSION THREAD

**Discussion by klajmajk** [dr-dobermann/gobpm#54] [General]
Hello @dr-dobermann ,

I found your project and I find it very interesting as we are working on something similar. What motivated us is the change of licence policy of Camunda. Our aim would be to create alternative BPMS which should be both easy to use and scalable. Feel free to check our project https://github.com/pbinitiative. We based this on https://github.com/nitram509/lib-bpmn-engine and https://github.com/rqlite/rqlite.

I would like to ask whether you would be interested to talk to us and discuss whether we could maybe cooperate.

Adam

**Comment by dr-dobermann** [comment]
Hi Adam,
I checked your project and found a lots of things you made very well. I believe all parties could have benefits from the cooperation.
My only concern is than I'm working on my project only in my free time and now I cannot support strict schedule in its development.

Ruslan

**Reply by klajmajk** [reply]
I completely understand this. Right now we are in simmilar mode. But we have big plans and right now allot of companies are interested in this, so raising some money for this activity seems quite doable. But we would like to get in touch with people that see things the same way we do so that we could collaborate. Please could you somehow share your email address so that we could arrange a call where we could discuss possible collaboration? (my email should be visible at my profile)
